### PR TITLE
Improve ping coloring in the server browser

### PR DESCRIPTION
**Pull request by Calinou**
There are now 5 possible colors instead of just 3. This makes for a smoother gradient.

### CODE REVIEW

File: src/engine/rendertext.cpp [hunk 1]
```diff
@@ -304,6 +305,7 @@ static void text_color(char c, bvec4 *stack, int size, int &sp, bvec4 &color, in
         case 'v': case 'V': stack[sp] = color = TVECA(c == 'V' ? colourdarkviolet : colourviolet, alpha); break;
         case 'p': case 'P': stack[sp] = color = TVECA(c == 'P' ? colourdarkpurple : colourpurple, alpha); break;
         case 'n': case 'N': stack[sp] = color = TVECA(c == 'N' ? colourdarkbrown : colourbrown, alpha); break;
+        case 'h': case 'H': stack[sp] = color = TVECA(c == 'H' ? colourdarkgreenyellow : colourgreenyellow, alpha); break;
```
Comment (Calinou) [inline src/engine/rendertext.cpp]: I picked `h` as a letter randomly, maybe there's a better choice?

Comment (molexted) [reply src/engine/rendertext.cpp]: Could use C for *chartreuse* if that's not conflicting with anything, I suppose.

Comment (Calinou) [reply src/engine/rendertext.cpp]: `c` is already used for cyan above :slightly_frowning_face:

Comment (qreeves) [reply src/engine/rendertext.cpp]: `H` seems fine, I'd also change the name `COLOURVAR` to `chartreuse`.

**Pull request merged**